end-to-end-tests/src/helpers/ctx.rs

```diff
@@ -240,7 +240,7 @@ pub async fn build_client() -> Result<oxide_client::Client> {
             })
         },
         &Duration::from_secs(1),
-        &Duration::from_secs(600),
+        &Duration::from_secs(1200),
```

Comment:
Can we revert the timeout here? It isn't necessary, right?

Reply:
ah right! will do